numerics/frequency_analysis_body.hpp

```diff
@@ -112,20 +112,35 @@ IncrementalProjection(Function const& function,
   CHECK(ω.has_value());
 
   std::vector<Series> basis;
+  // The Poisson series basis[k] belongs to the subspace basis_subspaces[k];
+  // this remains true after orthonormalization, i.e., q[k] belongs to the
+  // subspace basis_subspaces[k] below.
+  std::vector<PoissonSeriesSubspace> basis_subspaces;
 
   int basis_size;
   if (ω.value() == AngularFrequency{}) {
```

Comment:
Please put a `TODO(phl)` to refactor this if statement and the one near line 170 in a separate function.